### PR TITLE
fixing enleak for error-forgot-password json mock

### PR DESCRIPTION
**Pull request by anipendakur-okta**
## Description:
image
image

## PR Checklist

- [ ] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Issue:

- [OKTA-389440](https://oktainc.atlassian.net/browse/OKTA-389440)

### CODE REVIEW

**Comment by anipendakur-okta** [comment]
Note - Related backend change - https://github.com/okta/okta-core/pull/52825/files

File: playground/mocks/config/responseConfig.js [hunk 1]
```diff
@@ -14,7 +14,8 @@ const idx = {
   ],
 
   '/idp/idx/introspect': [
-    'identify',
+    'error-forgot-password',
```
Comment (nikhilvenkatraman-okta) [inline playground/mocks/config/responseConfig.js:17]: Lets revert this change

**Review by nikhilvenkatraman-okta — APPROVED** [review]

File: packages/@okta/i18n/src/properties/login.properties [hunk 2]
```diff
@@ -966,6 +967,7 @@ oie.user.profile.firstname= First name
 oie.user.profile.primary.email= Primary email
 oie.selfservice.unlock_user.success.message = You can log in using your existing username and password.
 oie.selfservice.unlock_user.failed.message = We are unable to unlock your account at this time, please contact your administrator
+oie.selfservice.user.unlock.not.allowed = Self Service Unlock is not allowed at this time. Please contact support for assistance.
```
Comment (pradeepdewda-okta) [inline packages/@okta/i18n/src/properties/login.properties:970]: we should keep the messaging consistent..
`Please contact support for assistance.`
`please contact your administrator` 

??

Comment (anipendakur-okta) [reply packages/@okta/i18n/src/properties/login.properties:970]:
```
oie.selfservice.unlock_user.failed.message = We are unable to unlock your account at this time, please contact your administrator
```
* Seems like an old / invalid key. When I search for `oie.selfservice.unlock_user.failed.message` in `okta-core`, it is not found.
* When I search for the message `We are unable to unlock your account at this time, please contact your administrator` in `okta-core` I see it being mapped to `selfservice.unlock_user.challenge.failed` and `oie.selfservice.unlock_user.challenge.failed` in `enduser-backend.properties` file. Perhaps, this is only for enduser app then?
* This raises another question on the validity of the mock - https://github.com/okta/okta-signin-widget/blob/master/playground/mocks/data/idp/idx/error-unlock-account.json - This mock seems outdated to me. * Regardless of that, this PR is intended to address the addition of new keys (messages existed prior to this, but keys did not) from https://github.com/okta/okta-core/pull/52825/files . I have asked a question on reusing/consolidating a key if it already existed on the PR and it seemed like they didn't and hence these are new keys.  Ref: https://github.com/okta/okta-core/pull/52825/files#r631479492
cc: @praveenprakash-okta

Comment (praveenprakash-okta) [reply packages/@okta/i18n/src/properties/login.properties:970]: That's correct. Even though both messages mean the same, the wording is different and the one for which I added the key already exists.

Comment (nikhilvenkatraman-okta) [reply packages/@okta/i18n/src/properties/login.properties:970]: Lets create 2 follow up JIRAss

1: To remove `oie.selfservice.unlock_user.failed.message` as it seems invalid and does not map to any key in the backend.

2:  For the backend to consolidate the messages and use oie.selfservice.user.unlock.not.allowed  since that was not done in the other PR.

Comment (anipendakur-okta) [reply packages/@okta/i18n/src/properties/login.properties:970]: Opened this for consolidating backend messages - https://oktainc.atlassian.net/browse/OKTA-396369

Comment (anipendakur-okta) [reply packages/@okta/i18n/src/properties/login.properties:970]: Once that's done here's the UI task - https://oktainc.atlassian.net/browse/OKTA-396372

Comment (anipendakur-okta) [reply packages/@okta/i18n/src/properties/login.properties:970]: For #2, I think we should be using `selfservice.user.unlock.not.allowed` (V1 key) since that's the direction from last SIW office-hours.

**Review by pradeepdewda-okta — APPROVED** [review]
👍
